spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/discovery/NacosDiscoveryClientConfiguration.java

```diff
@@ -51,12 +52,28 @@ public DiscoveryClient nacosDiscoveryClient(
 		return new NacosDiscoveryClient(nacosServiceDiscovery);
 	}
 
+	/**
+	 * NacosWatch is no longer enabled by default .
+	 * see https://github.com/alibaba/spring-cloud-alibaba/issues/2868
+	 */
 	@Bean
 	@ConditionalOnMissingBean
-	@ConditionalOnProperty(value = "spring.cloud.nacos.discovery.watch.enabled", matchIfMissing = true)
+	@ConditionalOnProperty(value = "spring.cloud.nacos.discovery.watch.enabled", matchIfMissing = false)
```

Comment:
`matchIfMissing` default value is `false`, so no need to specify

Reply:
In general, I prefer to display specified values for better readability

Reply:
make sense